packages/framework/view-adapters/src/htmlview/htmlViewAdapter.tsx

```diff
@@ -34,11 +35,14 @@ export class HTMLViewAdapter implements IFluidHTMLView {
      * the React case.  This also doubles as a way for us to know if we are mounted or not.
      */
     private containerNode: HTMLElement | undefined;
+    private readonly view: FluidObject<IFluidHTMLView>;
```

Comment:
I think I might prefer keeping this as `FluidObject` here, and do the cast to `FluidObject<IFluidHTMLView>` when used in `render()` instead, just as for how it reads.